documentation/docs/discovery.mdx

```diff
@@ -1,127 +1,106 @@
 ---
 id: discovery
-title: Discovery
+title: Account Discovery
 ---
 
-When an entity wishes to interact with a User in Open Payments, they first require the Payment Pointer for that User.
-A Payment Pointer they will resolve to a Payment Pointer URL. This allows the entity to fetch the metadata for the server described
-by that URL as defined in [RFC8414](https://tools.ietf.org/html/rfc8414).
+import { Mermaid } from './Mermaid'
+import { PersonToPerson } from './personToPerson'
+import { RequestToPay } from './requestToPay'
 
-## Issuer and Subject
+At the heart of all interactions in Open Payments is an [account](./accounts).
+The account could be either the sending or the receiving side of the
+transaction. The entity initiating the transaction is the Open Payments
+[client](./terminology#client) and discovers the account at the Open Payments
+[server](./terminology#server).
 
-Given a URL that has been resolved from a Payment Pointer a client can derive
-both the issuer identifier and subject identifier. 
+In the following example of a [person-to-person payment](./p2p) from Alice to
+Bob, Alice's wallet is the client and therefor her wallet discovers Bob's
+account at his wallet to begin the transaction.
 
-Payment Pointers resolve to HTTPS URLs which are the unique subject identifiers
-for the subject of the Payment Pointer.
+<PersonToPerson />
 
-The Issuer identifier is the same URL with the `path`, `query string` and
-`fragment` trimmed away.
+However, in the second example Alice is the payee but her wallet still uses
+Bob's Payment Pointer to discover his account and create and authorize a mandate
+to pull funds from it.
 
-E.g. Given the Payment Pointer `$wallet.example/alice` the subject identifier is
-`https://wallet.example/alice` and the issuer identifier is
-`https://wallet.example`.
+<RequestToPay />
 
-Once the client has determined the issuer identifier it can retrieve the server
-metadata for the issuer.
+Discovery starts with an identifier, which in Open Payments is an account
+identifier called a **Payment Pointer**.
```

Comment:
```suggestion
identifier (typically a **Payment Pointer**).
```

Reply:
Deferring until we have alternative identifiers proposed